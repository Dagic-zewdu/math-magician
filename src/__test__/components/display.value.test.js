import React from 'react';
import renderer from 'react-test-renderer';
import Display from '../../components/calculator/display.value';

describe('should render without problems', () => {
  it('should render the display calculator', () => {
    const tree = renderer.create(<Display next="1" operation="+" total={100} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
