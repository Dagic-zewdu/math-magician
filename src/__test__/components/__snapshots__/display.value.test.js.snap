// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without problems should render the calculator 1`] = `
<div
  className="g-item gray"
  style={
    Object {
      "fontSize": "4rem",
      "gridColumn": "1/5",
      "justifyContent": "flex-end",
    }
  }
>
  1
</div>
`;

exports[`should render without problems should render the display calculator 1`] = `
<div
  className="g-item gray"
  style={
    Object {
      "fontSize": "4rem",
      "gridColumn": "1/5",
      "justifyContent": "flex-end",
    }
  }
>
  1
</div>
`;
